sessions/4/alexander_osetrov/rates.py

```diff
@@ -0,0 +1,87 @@
+import json
+from re import match
+from sys import exit
+
+
+currency_data = "./exchange_rate.json"
+
+
+def parse_new_rate(rate):
+    if match(r"[A-Z]{3}/[A-Z]{3}=.+", rate):
+        currency, value = rate.split("=")
+        try:
+            new_rate = {currency: float(value)}
+            return new_rate
+        except ValueError:
+            print(f"Invalid currency rate value: {value}. Number is expected")
+    else:
+        print(f"Invalid currency rate format: {rate}.\n"
+              f"Expected format: [Currency 1/Currency 2]=[Rate], e.g. GBP/EUR=1.11791")
+
+
+def parse_convert_request(convert_request):
+    if match(r"\d+(\.\d+)?\s[A-Z]{3}$", convert_request):
```

Comment:
I think for not regex experts it would be useful to either add comment what this regex expression does or separate it into a variable with descriptive name.